src/models/geograph.py

```diff
@@ -189,6 +190,11 @@ def crs(self):
         """Return crs of dataframe."""
         return self.df.crs
 
+    @property
+    def bounds(self):
+        """Return bounds of entire graph"""
+        return self.df.total_bounds
```

Comment:
```suggestion
        return self.df.sindex.bounds
```

Reply:
Minor point, but it seems a lot faster 
![image](https://user-images.githubusercontent.com/13484097/111698567-e5310400-882e-11eb-9cf4-5a6cf89e2dde.png)

Reply:
Nice, thank you!